{{ cookiecutter.dir_name }}/buildSrc/src/main/groovy/VocAndroidPlugin.groovy

```diff
@@ -0,0 +1,49 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+
+class VocPlugin implements Plugin<Project> {
+    void apply(Project project) {
+        project.extensions.create("voc", VocPluginExtension)
+
+        project.task('vocBuild') {
```

Comment:
A stylistic thing - can we simplify this name? If I'm reading this right, we're registering commands that we can execute as `./gradle build XXX` - if that's the case, I'd prefer to keep this as simple as `./gradle build compile` and `./gradle build run`, rather than referencing VOC etc. 

Similarly, this is a Python project, rather than VOC project - so `PythonPlugin` would seem a better name than `VocPlugin`.